src/main/java/org/alfresco/repo/content/transform/TikaPoweredContentTransformer.java

```diff
@@ -90,6 +91,8 @@
     protected static final String LINE_BREAK = "\r\n";
     public static final String WRONG_FORMAT_MESSAGE_ID = "transform.err.format_or_password";
     
+    private boolean extractBookmarksText = true;

```

Comment:
This instance variable (and all the code that uses it) should be in the sub class PdfBoxContentTransformer.
This will mean that you need to override the transformRemote method just like ArchiveContentTransformer does.